src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,94 @@
+import Link from 'next/link';
+import { ReactElement } from 'react';
+import Image from 'next/image';
+import { FooterNavList } from '../FooterNavList';
+
+interface FooterProps {
+  branchesTotal: number;
+  countriesTotal: number;
+  capitalsTotal: number;
+  footerMainTitle: string;
+  footerServicesTitle: string;
+  footerInfoTitle: string;
+  footerMainList: any;
```

Comment:
any is forbidden in 90+ percent of usages. I think it is an array of something, check the API and create the required types